### DISCUSSION TITLE
Welcome to http4s Discussions!

### DISCUSSION THREAD

**Discussion by rossabaker** [http4s/http4s#3999] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. We follow the [Scala code of conduct](https://http4s.org/code-of-conduct/).

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ChristopherDavenport** [comment]
:wave: Hi I'm Christopher Davenport, but I go by Chris.

I help maintain libraries, including http4s, and work extensively with the functional programming and typelevel libraries.

If I can ever be of any assistance, please let me know.(Except for Ross, he'll put me to work on boring things)
